app/models/organization.rb

```diff
@@ -83,6 +84,11 @@ def self.with_paid_feature_slugs(slugs)
     where("paid_feature_slugs ?& array[:keys]", keys: matching_slugs)
   end
 
+  # Organization kinds creatable by non-superadmins
+  def self.creatable_kinds
+    kinds.reject { |kind| kind == "ambassador" }
```

Comment:
Just a question (not a request for changing): Is this more performant that `kinds - ["ambassador"]`?

Reply:
Not sure. There's an extra array allocation with the array subtraction approach, but I think it's a negligible difference.

Semantically array subtraction amounts to 

```ruby
kinds.reject {|kind| lhs_arr.any? { |e| kind.eql?(e) } }
```

`reject` is a little more flexible since it accepts a block, array subtraction a little more succinct.